pulsar-client-cpp/pkg/rpm/SPECS/pulsar-client.spec

```diff
@@ -52,7 +52,7 @@ static library.
 
 %build
 cd pulsar-client-cpp
-cmake . -DBUILD_TESTS=OFF -DLINK_STATIC=ON
+cmake . -DBUILD_TESTS=OFF -DLINK_STATIC=ON -DPYTHON_INCLUDE_DIR=/usr/include/python2.7 -DPYTHON_LIBRARY=/usr/lib64
```

Comment:
The RPM shouldn't build Python extension, since the Python wheels are built and distributed on their own.

Rather, we should add `-DBUILD_PYTHON_WRAPPER=OFF`